src/game/serialization.rs

```diff
@@ -126,8 +126,21 @@ impl Encode for PostFlopGame {
         // version
         VERSION_STR.to_string().encode(encoder)?;
 
+        // Update state based on target storage whenever the state is solved
+        // TODO(Jacob): May be better to add is_solved() function onto state
```

Comment:
@JacobVanGeffen can we resolve this TODO?

Reply:
Yes, but I believe we can use `state.is_partially_solved` instead of the inequality below.